src/test/kotlin/vendor/EthUsageTest.kt

```diff
@@ -0,0 +1,23 @@
+package vendor
+
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+import org.web3j.protocol.Web3j
+import org.web3j.protocol.http.HttpService
+
+class EthUsageTest {
+
+    @Test
+    fun ethUsage() {
+        try {
+            val web3 = Web3j.build(HttpService())
+            web3.web3ClientVersion().observable().subscribe { x ->
+                val clientVersion = x.getWeb3ClientVersion()
+                print(clientVersion)
```

Comment:
What if client is running?
Should it assert with true or false?

Reply:
This test checks the library is loaded. So, it should always pass.

Reply:
Maybe it is better to add explicit `assert(true)` after printing client version?